e2e/cypress.json

```diff
@@ -11,5 +11,7 @@
   "ignoreTestFiles": ["**/__snapshots__/*", "**/__image_snapshots__/*"],
   "projectId": "ed7hed",
   "defaultCommandTimeout": 10000,
-  "taskTimeout": 60000
+  "taskTimeout": 60000,
+  "chromeWebSecurity": false,
```

Comment:
This is needed by the `cy.visit` command to work properly (that command was extended in e2e/support/commands.js).